DESCRIPTION

```diff
@@ -11,7 +11,7 @@ Description: A set of Study Data Tabulation Model (SDTM) datasets from the Clini
     package.
```

Comment:
Change to: inside the pharmaverse family of packages

Reply:
updated